infrastructure/cron-jobs/rerun-enrollee-application-rules.cron.yml

```diff
@@ -8,83 +8,75 @@ metadata:
     tags: cronjob
   name: rerun-enrollee-application-rules-cronjob-template
 objects:
-  - apiVersion: batch/v1beta1
-    kind: CronJob
-    metadata:
-      name: '${CRON_NAME}'
-    spec:
-      concurrencyPolicy: Forbid
-      jobTemplate:
-        spec:
-          template:
-            spec:
-              containers:
-                - command:
-                  - bash
-                  - '-c'
-                  - >-
-                    echo -e "-------- STARTING CRON --------\n"
+- apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: '${CRON_NAME}'
+  spec:
+    concurrencyPolicy: Forbid
+    jobTemplate:
+      spec:
+        template:
+          spec:
+            containers:
+            - command:
+              - bash
+              - '-c'
+              - >-
+                echo -e "-------- STARTING CRON --------\n" #magic___^_^___line echo -e "-------- Getting access_token --------\n" #magic___^_^___line TOKEN=$(curl -X POST "${KEYCLOAK_URL}/realms/${KEYCLOAK_REALM}/protocol/openid-connect/token" \
```

Comment:
What is this about `#magic___^_^___line`?

Reply:
I'm not sure, and it looks like this job isn't currently deployed. Let's leave this for the time being.